.ansible-lint

```diff
@@ -0,0 +1,9 @@
+---
+verbosity: 1
+quiet: False
+parseable: True
+use_default_rules: True
+rulesdir:
+  - $HOME/.ansible/ansible-lint-rules
+skip_list: []
+#   - '306'
```

Comment:
Please leave a note why you commented this out, or remove it.

Reply:
done.